config_flow.py

```
@@ -42,32 +59,108 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+        errors: dict[str, str] = {}
+        
+        if user_input is not None:
+            data = {
+                CONF_HOST: self._host,
+                CONF_AUTH_METHOD: AUTH_METHOD_SESSION_ID,
+                CONF_SESSION_ID: user_input[CONF_SESSION_ID],
+            }
+            
+            try:
+                info = await validate_input(self.hass, data)
             except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)